jquery_dom_selector_practice/js/exercise.js

```diff
@@ -85,7 +106,10 @@
  * Bonus: Try to do this with only one jQuery call. $("selector").css({attribute: "value", attribute: "value"});
  */
 
-
+$("#button3").click(question9);
+	function question9(){
+		$('p').css('font-family','Georgia').css('color', 'blue');
```

Comment:
Commas are needed between each "key/value" pairing with exception to the last one. That one, you can omit the trailing comma. This is why you don't need a comma after `'blue'`

```
$('p').css({
   'font-family': 'Georgia',
   'backgroundColor': 'green',
   'marginTop': '20px',
   'color': 'blue'
})
```